packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation_objc/QueueUtils.m

```diff
@@ -13,3 +13,7 @@ void FLTEnsureToRunOnMainQueue(dispatch_block_t block) {
     block();
   }
 }
+
+void FLTDispatchQueueSetSpecific(dispatch_queue_t queue, const void *key) {
```

Comment:
This function is moved from `QeueueTestUtils`. We now need it in `CameraPlugin` to access the ObjC dispatch queue API from Swift. It will be completely removed once `FLTCam` is migrated